di_website/templates/404-static.html

```diff
@@ -0,0 +1,128 @@
+        grunticon(["/assets/icons/ico.data.svg.css", "/assets/icons/ico.data.png.css", "/assets/icons/ico.fallback.css"]);
+    </script>
+    <noscript><link href="/assets/icons/ico.fallback.css" rel="stylesheet"></noscript>
+    <script src="/assets/js/libs/modernizr.js"></script>
+    <link rel="icon" type="image/png" href="/assets/assets/favicons/favicon-16x16.png" sizes="16x16">
+</head>
+<body class="body body--error" id="body-error">
+	<div class="ui-base">
+		<a href="#pagecontent" class="skiplink" data-js="skiplink">Skip to main content</a>
+		<header role="banner" class="header">
+            <div class="row">
+                <a href="/" class="branding">Development Initiatives</a>
+                <button role="button" class="navigation-primary-toggle" id="navigation-primary-toggle">
+                    <span>Menu</span>
+                </button>
+                <nav class="navigation-primary" role="navigation" id="navigation-primary">
+                    <ul class="navigation-primary__items">
+                        <li id="section-Who we are" class='navigation-primary__item'><a href="#">Who we are</a></li>
```

Comment:
Ah yeah, forgot to mention @davidebukali that the static error pages need to be very bare-boned. They're rare to come up (NGINX rarely has a 500 error on its own), so you can do with a simple styled message with some hardcoded CSS.

Reply:
@akmiller01 I used the link you shared at [Digital Ocean](https://www.digitalocean.com/community/tutorials/how-to-configure-nginx-to-use-custom-error-pages-on-ubuntu-14-04) to setup bare bones pages for nginx;

For wagtail errors i created a module called error;

To test set Debug = False and run server with --insecure flag because when Debug = False runserver does not handle static files and throws an error;

`ValueError: Missing staticfiles manifest entry for 'css/screen.min.css'  "GET /error-pagep/ HTTP/1.1" 500 59; `

Please see [this link](https://stackoverflow.com/questions/5836674/why-does-debug-false-setting-make-my-django-static-files-access-fail)

There is possibly a better solution to this using the original 

`STATICFILES_STORAGE = 'django.contrib.staticfiles.storage.ManifestStaticFilesStorage'`

 but so far this is what I have found with 

`STATICFILES_STORAGE = 'django.contrib.staticfiles.storage.StaticFilesStorage'`